src/index.js

```diff
@@ -124,20 +126,21 @@ class ForEachPlugin {
 					}, Array.isArray(template) ? [] : {});
 
 					const pathIsArray = path.match(/^(.+)\[(\d+)\]$/);
+					const { [key]: $forEach, ...result } = obj; // eslint-disable-line no-unused-vars
+
+					if (Array.isArray(interpolated) && Object.keys(result).length > 0) {
```

Comment:
Isn't this the same?
```suggestion
					if (Array.isArray(interpolated) && Object.keys(result).length) {
```

Reply:
Some people argue that `> 0` makes it more readable